acestep/gradio_ui/events/generation_handlers.py

```
@@ -737,16 +758,85 @@ def convert_src_audio_to_codes_wrapper(dit_handler, src_audio):
+        result.timesignature,   # time_signature
+        True,                   # is_format_caption
+    )
+
+
 def update_instruction_ui(
     dit_handler,
     task_type_value: str, 
     track_name_value: Optional[str], 
     complete_track_classes_value: list, 
-    audio_codes_content: str = "",
     init_llm_checked: bool = False,
```

Ruff (0.15.0)

[warning] Unused function argument: `init_llm_checked`

(ARG001)